tests/tripExistence.spec.ts

```diff
@@ -1,7 +1,23 @@
-import { test } from './utils'
+import { test, expect, urlMatcher } from './utils'
 
-test('choosing params in "קיום נסיעות" and organize by date/hour ', async ({ page }) => {
+test.beforeEach(async ({ page, advancedRouteFromHAR }) => {
+  advancedRouteFromHAR('tests/HAR/menu.har', {
+    updateContent: 'embed',
+    update: false,
+    notFound: 'abort',
+    url: /stride-api/,
+    matcher: urlMatcher,
+  })
```

Comment:
That's so cool, I'm so glad to see you're using it! How was your experience?
By the way, feel free to record a separate HAR file for this file:
```suggestion
  advancedRouteFromHAR('tests/HAR/tripExistence.har', {
    updateContent: 'embed',
    update: true,
    notFound: 'abort',
    url: /stride-api/,
    matcher: urlMatcher,
  })
```
After running one test with this version, `update` can be switched to back `false`.

(Only if you want to try it out, don't feel obligated)

Reply:
Thanks! I'm playing with some trial and error and I've made all sorts of mistakes here :) but I seem to have finally fixed it, and also made a new HAR as you wrote, and pushed it :)
Thanks!!

Reply:
That's so exciting! I'm truly happy to see it. If you have any feedback regarding the developer experience, I would really like to improve things were possible. 
The [advanced har library](https://github.com/NoamGaash/playwright-advanced-har) is also a project that I maintain so please feel comfortable leaving feedback and ideas and we can improve it as well.


<img src="https://media1.giphy.com/media/oRFdoOGBqFikXk8PXh/giphy.gif"/>